### PR TITLE
pancan spanish translations

### PR DESCRIPTION
**Pull request by ssettipalli**
pancan spanish translations

### CODE REVIEW

File: study-builder/studies/pancan/i18n/es.conf [hunk 1]
```diff
@@ -1,969 +1,969 @@
...
+  "summary_inprogress": "[es] Please complete providing your child's email so we can contact them about the upcoming re-consent process.",
+  "summary_complete": "[es] Thank you for providing us with your child's email so we can contact them about the upcoming re-consent process.",
+  "intro": "[es] In a few months, your child will reach the age when they will need to sign a consent form to continue participation in the study. Please provide us with your child's email so we can contact them about the upcoming re-consent process.",
+  "email_prompt": "[es] Your child's email address",
+  "email_confirm_prompt": "[es] Confirm your child's email address",
+  "email_mismatch_msg": "[es] Please make sure your child's email address matches",
   "email_placeholder": "name@email.com",
-  "email_req_hint": "Please provide an email address to contact your child.",
+  "email_req_hint": "[es] Please provide an email address to contact your child.",
 },
```
Comment (evgeniipr) [inline study-builder/studies/pancan/i18n/es.conf:776]: These should left with [es] prefixes?

File: study-builder/studies/pancan/i18n/es.conf [hunk 2]
```diff
@@ -1,969 +1,969 @@
...
+  "name_prompt": "Su Nombre",
+  "child_name_prompt": "Nombre de su hijo",
+  "child_dob_prompt": "Fecha de nacimiento de su hijo",
+  "child_address_title": "Dirección postal de su hijo",
+  "err_dob_need_self": "[es] Please check the date of birth you entered for your child. If the date is correct, then your child has reached the age of majority and must self-enroll in order to participate in this study. If you think there is a mistake, please reach out to us at <a href=\"tel:651-403-5315\" class=\"Link\">651-403-5315</a> or <a href=\"mailto:info@joincountmein.org\" class=\"Link\">info@joincountmein.org</a>.",
+  "err_dob_need_assent": "[es] Please check the date of birth you entered for your child. If the date is correct, then along with your consent, your child must give their own assent to participate in the study. In that case, please restart the process with a different email address, or contact us for help at <a href=\"tel:651-403-5315\" class=\"Link\">651-403-5315</a> or <a href=\"mailto:info@joincountmein.org\" class=\"Link\">info@joincountmein.org</a>.",
```
Comment (evgeniipr) [inline study-builder/studies/pancan/i18n/es.conf:655]: Above 2 also with [es] prefixes.

**Review by evgeniipr — APPROVED** [review]

File: study-builder/studies/pancan/i18n/es.conf (hunk 1 above)
Comment (ssettipalli) [reply study-builder/studies/pancan/i18n/es.conf:776]: translation not provided  hence [es]

File: study-builder/studies/pancan/i18n/es.conf (hunk 2 above)
Comment (ssettipalli) [reply study-builder/studies/pancan/i18n/es.conf:655]: translation not provided hence [es]

**Pull request merged**